wodles/aws/aws_s3.py

```diff
@@ -3462,6 +3472,10 @@ def arg_valid_regions(arg_string):
     final_regions = []
     regions = arg_string.split(',')
```

Comment:
The case where duplicate regions are declared should be considered

Reply:
Fixed in c7b2de4